.github/workflows/lib-injection.yml

```diff
@@ -1,11 +1,6 @@
 name: "Library Injection"
 on:
   push:
-    branches:
```

Comment:
For [some reason](https://github.com/actions/checkout/blob/v3.0.2/README.md#checkout-pull-request-head-commit-instead-of-merge-commit) using the `pull_request` event results in some merge commit being used for the `${{ github.sha}}` instead of the actual commit that triggered the workflow.

This causes a mismatch between the image published in the workflow and the gitlab job which uses `$CI_COMMIT_SHA`.


This job is cheap enough that it should be ok to run on each `push`.